.iyarc

```diff
@@ -1,2 +1,3 @@
 # improved-yarn-audit advisory exclusions
 GHSA-257v-vj4p-3w2h
+GHSA-6fc8-4gx4-v693
```

Comment:
The reason is i added the '--recursive' operator to improved-yarn-audit's flags which checks transitive properties too. if i run `yarn npm audit` without the `--recursive` flag this goes away. I compared the `yarn why ws` on both develop and here and there is no difference (to my eye, the output is very different but thats formatting). I think it has to be the way in which the `--recursive` flag works.

Reply:
The audit command we were using with Yarn v1 was already recursive. We weren't using that flag only because it was _always_ recursive, with no way to choose otherwise.

Reply:
I would say the changes in the node_modules linking strategy but that shouldn't result in different versions of dependencies being used. More than likely its a mix of the existing improved yarn audit issue you pointed out in slack, my patch to improved yarn audit and the change from yarn v1 to v3.